bot/cogs/sticky.py

```diff
@@ -0,0 +1,256 @@
+    @app_commands.describe(channel="Target Channel")
+    async def remove_sticky_message(
+        self,
+        interaction: Interaction,
+        channel: Union[discord.TextChannel, discord.ForumChannel, discord.Thread]
+    ) -> None:
+        await interaction.response.defer()
+        if interaction.permissions.manage_channels:
+            async with self.db_pool.acquire() as conn:
+                res = await conn.fetch(
+                    "SELECT channel_id,message_id FROM sticky WHERE channel_id = $1;", channel.id
+                )
+            if not res:
+                embed = discord.Embed(
+                    color=discord.Color.red(),
+                    title="❌ Sticky message not exist",
+                    description=f"Tidak ada sticky message pada channel <#{channel.id}>",
+                    timestamp=datetime.now()
+                )
+            else:
+                data = dict(res[0])
+
+                sticky = await channel.fetch_message(data['message_id'])
+                await sticky.delete()
+
+                async with self.db_pool.acquire() as conn:
+                    await conn.execute(
+                        "DELETE FROM sticky WHERE channel_id = $1;", channel.id
+                    )
+
+                embed = discord.Embed(
+                    color=discord.Color.green(),
+                    title="✅ Sticky message removed successfully",
+                    description=f"Berhasil menghapus sticky message pada channel <#{channel.id}>",
+                    timestamp=datetime.now()
+                )
+        else:
+            embed = discord.Embed(
+                color=discord.Color.red(),
+                title="❌ You Don't Have Permission To Delete Sticky Message",
+                description=f"Permission Manage Channel Dibutuhkan",
+                timestamp=datetime.now()
+            )
+
+        embed.set_footer(
+            text=f"{str(interaction.user)}",
+            icon_url=interaction.user.display_avatar.url
+        )
+
+        await interaction.followup.send(embed=embed)
+
+    @commands.guild_only()
+    @app_commands.command(name="purge", description="Remove all sticky message from channels")
+    async def purge_sticky_message(
+        self,
+        interaction: Interaction
+    ) -> None:
+        await interaction.response.defer()
+        if interaction.permissions.manage_channels:
+            async with self.db_pool.acquire() as conn:
+                res = await conn.fetch(
+                    "SELECT * FROM sticky;"
+                )
+                data = [dict(row) for row in res]
+                for sticky in data:
+                    channel = await self.bot.fetch_channel(sticky['channel_id'])
+                    message = await channel.fetch_message(sticky['message_id'])
+                    await message.delete()
+                
+            async with self.db_pool.acquire() as conn:
```

Comment:
```suggestion
            
```
Remove this line for computation efficiency. No need to call `db_pool.acquire()` twice in this function.